ts_compiler.opam

```diff
@@ -13,6 +13,7 @@ bug-reports: "https://github.com/gvorvik/ts-compiler/issues"
 depends: [
   "ocaml" {>= "5.0.0"}
   "dune" {>= "3.14"}
+  "bisect_ppx" {>= "2.5.0"}
```

Comment:
Refer to #8. I added a makefile to make running the code coverage thing more ergonomic